content/english/android-tictactoe/booleans.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
FYI For these "basics" concepts (e.g. booleans, comments) we will be building a Java Basics curriculum - we'll take some time after the hackathon to merge these pages into the basics workshop (so this workshop will just have the activities for building the TicTacToe game).